aws_adfs/_duo_authenticator.py

```diff
@@ -247,6 +247,7 @@ def _authentication_result(
             )
         )
     result_url = response.json()['response']['result_url']
+    sid = response.json()['response']["sid"]
```

Comment:
You may want to test for the existence of `sid` in the response before trying to access it.

Also, please use single quotes.